apis/certificatemanager/v1alpha1/types.generated.go

```diff
@@ -14,6 +14,41 @@
 
 package v1alpha1
 
+// +kcc:proto=google.cloud.certificatemanager.v1.TrustConfig.IntermediateCA
+type TrustConfig_IntermediateCA struct {
+	// PEM intermediate certificate used for building up paths
+	//  for validation.
+	//
+	//  Each certificate provided in PEM format may occupy up to 5kB.
+	// +kcc:proto:field=google.cloud.certificatemanager.v1.TrustConfig.IntermediateCA.pem_certificate
+	PemCertificate *string `json:"pemCertificate,omitempty"`
```

Comment:
Is this the actual pem certificate data or a path to the pem file? If it's the certificate, this is sensitive data and should be stored somewhere else. Suggest changing this to k8s secret